API/chat/views.py

```diff
@@ -45,7 +45,7 @@ def get(self, request, type, target, *args, **kwargs):
         channel = get_object_or_404(Channel, name=target)
 
         messages = Message.objects.values(
-            'text', 'username', 'datetime_start', 'typing', 'id', 'datetime_sent'
+            'text', 'username', 'datetime_start', 'typing', 'id', 'datetime_sent', 'message_type'
```

Comment:
Calling the new field 'type' will be confusing in regards to the target type which is currently 'channel' (see POST form above in which you have a 'type' which is not the same thing!). We should call it something else.